doc/services/device_mgmt/smp_groups/smp_group_2.rst

```diff
@@ -74,12 +76,12 @@ Statistics group data response header:
 
 CBOR data of successful response:
 
-.. code-block:: none
+.. code-block:: cddl
 
     {
-        (str)"name"     : (str)
-        (str)"fields"   : {
-            (str)<entry_name> : (uint)
+        "name"     : tstr,
+        "fields"   : {
+            (tstr)<entry_name> : uint,
```

Comment:
maybe drop the angle brackets as their semantics in CDDL is for generics?

Reply:
Not really sure how to represent this because "entry name" is a description of it, not what it would actually be, and there can be many of them. Don't think this can be properly represented in cddl unless someone knew the exact ones they were receiving?

Reply:
Also ping on this

Reply:
ya sorry. I can definitely live with the fact that this would be "kind of CDDL" syntax, so feel free to just leave as-is. Another option that might work would maybe be:

```suggestion
            (tstr)stat_1 : uint,
            (tstr)stat_2 : uint,
```

as it makes it clearer that it cannot possibly be actual stat names and are just placeholders.
Anyhow, your call, and no strong opinion!